ops-catalog/template-common.flow.yaml

```diff
@@ -0,0 +1,65 @@
+collections:
+  ops.DATAPLANE/logs:
+    schema: ops-log-schema.json
+    key: [/shard/name, /shard/keyBegin, /shard/rClockBegin, /ts]
+    projections:
+      kind:
+        location: /shard/kind
+        partition: true
+      name:
+        location: /shard/name
+        partition: true
+
+  ops.DATAPLANE/stats:
+    schema: ops-stats-schema.json
+    key: [/shard/name, /shard/keyBegin, /shard/rClockBegin, /ts]
+    projections:
+      kind:
+        location: /shard/kind
+        partition: true
+      name:
+        location: /shard/name
+        partition: true
+
+  ops.DATAPLANE/catalog-stats-L1:
+    schema: ops-catalog-stats-schema.json
+    key: [/catalogName, /grain, /ts]
+
+    derivation:
+      shards:
+        minTxnDuration: 5s
+      typescript: { module: "catalog-stats.ts" }
+      transform:
+        logs:
+          source:
+            name: ops.DATAPLANE/logs
+          publish: { lambda: typescript }
+        stats:
+          source:
+            name: ops.DATAPLANE/stats
+          publish: { lambda: typescript }
+
+  ops/catalog-stats-L2:
```

Comment:
perhaps just also have this be in ops.gcp-whatever ?

I don't believe a separate `ops/` tenant makes sense anymore -- we'll pick an arbitrary data-plane and not others to run it in, and the one we have seems reasonable.

Reply:
Changed it to all use the same gcp-specific tenant